schema/amber-endorsement/v0.1/endorsement_test.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// TODO(#23): Move this to the `amber` package.
+package schema
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"testing"
+
+	"github.com/xeipuuv/gojsonschema"
+)
+
+func TestExampleAmberEndorsement(t *testing.T) {
+	schemaPath := "statement.json"
+	examplePath := "example.json"
+
+	schemaLoader, err := loadJSON(schemaPath)
+	if err != nil {
+		t.Fatalf("Couldn't load schema file %v: %v", schemaPath, err)
+	}
```

Comment:
I do low key wonder what the error ends up looking, since `loadJSON` and this add a description :)

Reply:
Here is an example:
```
Couldn't load schema file statements.json: couldn't read json file statements.json: open statements.json: no such file or directory
```

I think, at least internally, it is conventional for each nested error to add a context-specific text.